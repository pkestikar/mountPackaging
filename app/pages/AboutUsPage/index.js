import React from 'react';
import Container from '@material-ui/core/Container';
import Grid from '@material-ui/core/Grid';
import { MQMaxMedium, MQMinLarge } from "../../utilities/DeviceQueries";

const AboutUsPage = (props) => {

    const ProcessComposer = (props) => {
        return (
            <div className="process-info-wrapper">
                <h2>{props.processTitle}</h2>
                <p>
                    {props.processDescription}
                </p>
                {props.showMobileArrow &&
                <MQMaxMedium>
                    <div className="arrow-wrapper__mobile">
                        <img src="/images/about-us/down-arrow.png" alt="down-arrow" />
                    </div>
                </MQMaxMedium>
                }
            </div>
        )
    }

    return (
        <div className="mp--about-us-page-container">
            <Container disableGutters>
                <Grid container className={'mp--page-title'}>
                    <Grid item xs={12}>
                        <h1>About Us</h1>
                    </Grid>
                </Grid>
                <Grid container className="director-info-container">
                    <Grid item xs={12} md={12} lg={4} className={'director-image-wrapper'}>
                        <img src="/images/about-us/director.png" alt="Director" />
                        <div className="director-info">
                            Achyut Kestikar, Director (Operations),
                            Mount Packaging Machinery Pvt Ltd.
                        </div>
                    </Grid>
                    <Grid item className="director-description-wrapper" xs={12} md={12} lg={8}>
                        <p>
                            Mount Packaging Machinery Pvt. Ltd. was established in 2005, based in Pune, India.
                        </p>
                        <p>
                            Over decades, we have developed a dedicated team of qualified engineers and technicians, who design the machines on high-end softwares, manufacture parts on automated CNC machines, built and assembled to global manufacturing practices.
                        </p>
                        <p>
                            Mount Packaging Machinery Pvt. Ltd. has worked with numerous small and large enterprises worldwide with domain knowledge of forth generation packaging technology. The machines are tested on factory floor, before they are dispatched to customers. Whether small or large, highest level of  safety standardsâ€™ are followed to international norms.
                        </p>
                        <p>
                            Current leadership: <br />
                            Mr. Achyut Kestikar, Director (Operations)
                            Achyut is a competent engineer with 30+ years of experience in packaging industry with a wide industrial experience in different functions. He has a proven record of building and managing large complex projects from concept to completion phase.
                        </p>
                    </Grid>
                </Grid>
            </Container>

            <div className="mp--our-process-container">
                <Container disabledGutters>
                    <Grid container justify={'center'} alignItems={'center'}>
                        <h2 className={'container-title'}>Our Process</h2>

                        <Grid container justify={'center'} className={'process-info-row'}>
                            <Grid item md={4}>
                                <ProcessComposer
                                    processTitle={'Study Requirements'}
                                    processDescription={'Once the customer is comfortable with the quotation, we revisit the customer requirements to prepare a list of high-level design goals and objectives.'}
                                    showMobileArrow
                                />
                            </Grid>
                            <MQMinLarge>
                                <Grid item md={4}>
                                    <div className="arrow-wrapper__desktop">
                                        <img src="/images/about-us/right-arrow.png" alt="right-arrow" />
                                    </div>
                                </Grid>
                            </MQMinLarge>
                            <Grid item md={4}>
                                <ProcessComposer
                                    processTitle={'Meeting with customer'}
                                    processDescription={'Meetups with our customer helps us present our initial design thoughts. If required, we go back to revise the requirements.'}
                                    showMobileArrow
                                />
                            </Grid>
                        </Grid>

                        <MQMinLarge>
                            <Grid container justify={'flex-end'} className={'process-info-row'}>
                                <Grid item md={4}>
                                    <div className="arrow-wrapper__desktop">
                                        <img src={'/images/about-us/down-arrow.png'} alt={'down-arrow'} />
                                    </div>
                                </Grid>
                            </Grid>
                        </MQMinLarge>

                        <Grid container justify={'center'} className={'process-info-row'}>
                            <Grid item md={4}>
                                <ProcessComposer
                                    processTitle={'Implement'}
                                    processDescription={'This is the phase where the design comes to life.'}
                                    showMobileArrow
                                />
                            </Grid>
                            <MQMinLarge>
                                <Grid item md={4}>
                                    <div className={'arrow-wrapper__desktop'}>
                                        <img src="/images/about-us/left-arrow.png" alt="left-arrow" />
                                    </div>
                                </Grid>
                            </MQMinLarge>
                            <Grid item md={4}>
                                <ProcessComposer
                                    processTitle={'Design'}
                                    processDescription={'Once the requirements are finalized, we get to the main step of designing.'}
                                    showMobileArrow
                                />
                            </Grid>
                        </Grid>

                        <MQMinLarge>
                            <Grid container justify={'flex-start'} className={'process-info-row'}>
                                <Grid item md={4}>
                                    <div className={'arrow-wrapper__desktop'}>
                                        <img src={'/images/about-us/down-arrow.png'} alt={'down-arrow'} />
                                    </div>
                                </Grid>
                            </Grid>
                        </MQMinLarge>

                        <Grid container className={'process-info-row'}>
                            <Grid item md={4} className={'process-info-item-wrapper'}>
                                <ProcessComposer
                                    processTitle={'Deliver'}
                                    processDescription={`After thorough testing, it's time to ship the customized packaging machine to your factory floor.`}
                                />
                            </Grid>
                        </Grid>
                    </Grid>
                </Container>
            </div>
        </div>
    )
};

export default AboutUsPage;
