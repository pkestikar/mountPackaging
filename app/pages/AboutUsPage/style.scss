.mp--about-us-page-container {
  h2 {
    margin-bottom: 0.5rem;
  }
  .mp--page-title {
    h1 {
      margin-left: 3rem;
    }
  }
  margin-top: $top-bar-height__large;
  @media only screen and (max-width: $medium) {
    margin-top: $top-bar-height__small;
  }

  .director-info-container {
    margin-bottom: 3.5rem;
    @media only screen and (max-width: $medium) {
      padding: 2rem;
    }
  }

  .director-image-wrapper {
    overflow: hidden;
    text-align: center;
    @media only screen and (max-width: $medium) {
      margin-bottom: 2rem;
    }
    img {
      height: 412px;
      width: 412px;
      object-fit: cover;
    }
    .director-info {
      margin-top: 0.5rem;
      text-decoration: underline;
    }
  }

  .director-description-wrapper {
    padding: 0 3rem;
    @include breakpoint(medium down) {
      padding: 0;
    }
  }

  .mp--our-process-container {
    background: var(--navy-blue);
    color: white;
    width: 100%;
    padding-bottom: 4rem;
    .container-title {
      margin: 2.5rem 0;
    }
    .process-info-row {
      padding: 0 5rem;
      min-height: 150px;
      @include breakpoint(small down) {
        padding: 0 3rem;
      }
      .arrow-wrapper__desktop {
        display: flex;
        justify-content: center;
        align-items: center;
        height: 100%;
      }
    }

    .process-info-wrapper {
      color: white;
      max-width: 350px;
      h2, p {
        text-align: center;
      }
      .arrow-wrapper__mobile {
        width: 100%;
        display: flex;
        justify-content: center;
        align-items: center;
        margin: 2rem 0;
        img {
          margin: 0 auto;
        }
      }
    }
    .process-info-item-wrapper {
      @include breakpoint(medium down) {
        margin: 0 auto;
      }
    }
  }
}