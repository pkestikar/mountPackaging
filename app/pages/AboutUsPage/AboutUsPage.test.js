import React from 'react';
import { shallow } from 'enzyme';
import toJSON from "enzyme-to-json";

import AboutUsPage from "Pages/AboutUsPage";

describe('AboutUsPage component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<AboutUsPage />);
    });

    it('renders correctly', () => {
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});
