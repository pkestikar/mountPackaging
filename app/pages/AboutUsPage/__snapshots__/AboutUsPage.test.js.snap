// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUsPage component renders correctly 1`] = `
<div
  className="mp--about-us-page-container"
>
  <WithStyles(ForwardRef(Container))
    disableGutters={true}
  >
    <WithStyles(ForwardRef(Grid))
      className="mp--page-title"
      container={true}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        xs={12}
      >
        <h1>
          About Us
        </h1>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      className="director-info-container"
      container={true}
    >
      <WithStyles(ForwardRef(Grid))
        className="director-image-wrapper"
        item={true}
        lg={4}
        md={12}
        xs={12}
      >
        <img
          alt="Director"
          src="/images/about-us/director.png"
        />
        <div
          className="director-info"
        >
          Achyut Kestikar, Director (Operations), Mount Packaging Machinery Pvt Ltd.
        </div>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        className="director-description-wrapper"
        item={true}
        lg={8}
        md={12}
        xs={12}
      >
        <p>
          Mount Packaging Machinery Pvt. Ltd. was established in 2005, based in Pune, India.
        </p>
        <p>
          Over decades, we have developed a dedicated team of qualified engineers and technicians, who design the machines on high-end softwares, manufacture parts on automated CNC machines, built and assembled to global manufacturing practices.
        </p>
        <p>
          Mount Packaging Machinery Pvt. Ltd. has worked with numerous small and large enterprises worldwide with domain knowledge of forth generation packaging technology. The machines are tested on factory floor, before they are dispatched to customers. Whether small or large, highest level of  safety standardsâ€™ are followed to international norms.
        </p>
        <p>
          Current leadership: 
          <br />
          Mr. Achyut Kestikar, Director (Operations) Achyut is a competent engineer with 30+ years of experience in packaging industry with a wide industrial experience in different functions. He has a proven record of building and managing large complex projects from concept to completion phase.
        </p>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Container))>
  <div
    className="mp--our-process-container"
  >
    <WithStyles(ForwardRef(Container))
      disableGutters={true}
    >
      <WithStyles(ForwardRef(Grid))
        alignItems="center"
        container={true}
        justify="center"
      >
        <h2
          className="container-title"
        >
          Our Process
        </h2>
        <WithStyles(ForwardRef(Grid))
          className="process-info-row"
          container={true}
          justify="center"
        >
          <WithStyles(ForwardRef(Grid))
            item={true}
            md={4}
          >
            <ProcessComposer
              processDescription="Once the customer is comfortable with the quotation, we revisit the customer requirements to prepare a list of high-level design goals and objectives."
              processTitle="Study Requirements"
              showMobileArrow={true}
            />
          </WithStyles(ForwardRef(Grid))>
          <MQMinLarge>
            <WithStyles(ForwardRef(Grid))
              item={true}
              md={4}
            >
              <div
                className="arrow-wrapper__desktop"
              >
                <img
                  alt="right-arrow"
                  src="/images/about-us/right-arrow.png"
                />
              </div>
            </WithStyles(ForwardRef(Grid))>
          </MQMinLarge>
          <WithStyles(ForwardRef(Grid))
            item={true}
            md={4}
          >
            <ProcessComposer
              processDescription="Meetups with our customer helps us present our initial design thoughts. If required, we go back to revise the requirements."
              processTitle="Meeting with customer"
              showMobileArrow={true}
            />
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
        <MQMinLarge>
          <WithStyles(ForwardRef(Grid))
            className="process-info-row"
            container={true}
            justify="flex-end"
          >
            <WithStyles(ForwardRef(Grid))
              item={true}
              md={4}
            >
              <div
                className="arrow-wrapper__desktop"
              >
                <img
                  alt="down-arrow"
                  src="/images/about-us/left-arrow.png"
                  style={
                    Object {
                      "transform": "rotate(-90deg)",
                    }
                  }
                />
              </div>
            </WithStyles(ForwardRef(Grid))>
          </WithStyles(ForwardRef(Grid))>
        </MQMinLarge>
        <WithStyles(ForwardRef(Grid))
          className="process-info-row"
          container={true}
          justify="center"
        >
          <WithStyles(ForwardRef(Grid))
            item={true}
            md={4}
          >
            <ProcessComposer
              processDescription="This is the phase where the design comes to life."
              processTitle="Implement"
              showMobileArrow={true}
            />
          </WithStyles(ForwardRef(Grid))>
          <MQMinLarge>
            <WithStyles(ForwardRef(Grid))
              item={true}
              md={4}
            >
              <div
                className="arrow-wrapper__desktop"
              >
                <img
                  alt="left-arrow"
                  src="/images/about-us/left-arrow.png"
                />
              </div>
            </WithStyles(ForwardRef(Grid))>
          </MQMinLarge>
          <WithStyles(ForwardRef(Grid))
            item={true}
            md={4}
          >
            <ProcessComposer
              processDescription="Once the requirements are finalized, we get to the main step of designing."
              processTitle="Design"
              showMobileArrow={true}
            />
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
        <MQMinLarge>
          <WithStyles(ForwardRef(Grid))
            className="process-info-row"
            container={true}
            justify="flex-start"
          >
            <WithStyles(ForwardRef(Grid))
              item={true}
              md={4}
            >
              <div
                className="arrow-wrapper__desktop"
              >
                <img
                  alt="down-arrow"
                  src="/images/about-us/left-arrow.png"
                  style={
                    Object {
                      "transform": "rotate(-90deg)",
                    }
                  }
                />
              </div>
            </WithStyles(ForwardRef(Grid))>
          </WithStyles(ForwardRef(Grid))>
        </MQMinLarge>
        <WithStyles(ForwardRef(Grid))
          className="process-info-row"
          container={true}
        >
          <WithStyles(ForwardRef(Grid))
            className="process-info-item-wrapper"
            item={true}
            md={4}
          >
            <ProcessComposer
              processDescription="After thorough testing, it's time to ship the customized packaging machine to your factory floor."
              processTitle="Deliver"
            />
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Container))>
  </div>
</div>
`;
